lib/parse/datatypes.rb

```diff
@@ -334,6 +334,11 @@ def save
       resp
     end
 
+    def delete
+      uri = Parse::Protocol.file_uri(@parse_filename)
+      resp = Parse.client.delete(uri)
```

Comment:
there's no point into memorizing the return value into a local variable as the last statement.